cli/src/pcluster/resources/head_node/user_data.sh

```diff
@@ -87,7 +85,7 @@ else
   error_exit "This AMI was not baked by ParallelCluster. Please use pcluster createami command to create an AMI by providing your AMI as parent image."
 fi
 if [ "${!custom_cookbook}" != "NONE" ]; then
-  curl --retry 3 -v -L -o /etc/chef/aws-parallelcluster-cookbook.tgz -z "$(cat /etc/chef/aws-parallelcluster-cookbook.tgz.date)" ${!cookbook_url}
+  curl --retry 3 -v -L -o /etc/chef/aws-parallelcluster-cookbook.tgz ${!cookbook_url}
```

Comment:
the presign url should be created with --region option which is same as cluster region, otherwise, we can't get the https 200

Reply:
After our discussion, add bucket_region and additional policy to retrieve the bucket region